docs/src/modules/components/DiamondSponsors.js

```diff
@@ -54,11 +55,27 @@ export default function DiamondSponsors(props) {
           href="https://octopus.com/?utm_source=materialui&utm_medium=referral"
           rel="noopener noreferrer sponsored"
           target="_blank"
-          style={{ width: 40, height: 40 }}
+          sx={{
+            width: 40,
+            height: 40,
+            pt: 0.7,
+            pl: 0.7,
+            border: `1px solid ${
+              theme.palette.mode === 'dark'
+                ? theme.palette.primaryDark[700]
+                : theme.palette.grey[200]
+            }`,
+            borderRadius: 1,
+            transition: theme.transitions.create(['color', 'border-color']),
+            '&:hover': {
+              borderColor: 'currentColor',
+              color: theme.palette.mode === 'dark' ? theme.palette.primary[300] : theme.palette.primary[500],
+            },
+          }}
```

Comment:
@oliviertassinari Hey! Quickly answering to the point you left on the other comment:

> Do we need the borders in? I personally feel that leveraging spacing would be enough to communicate order:

Although having them without borders would work fine, I find the container to be helpful to bound the logos and not have them being floating around. It also draws better from the idea of "available slots" almost like little cartridges. Is it impeditive for you to keep them placed in the containers?!

Reply:
@danilo-leal I didn't notice this thread, sorry for the duplicate. I didn't get the question. Happy to hear other's perspectives on this.

I was sharing the feeling I got when browsing the side nav. I personally enjoy this UX more https://vuejs.org/v2/guide/

<img width="170" alt="Screenshot 2021-08-31 at 00 28 08" src="https://user-images.githubusercontent.com/3165635/131414015-800e7a1b-710e-415f-a563-0e9a5925df3b.png">

Reply:
Actually, now that I think about this problem, it makes me think about what I saw a year ago on https://lusaxweb.github.io/vuesax/development/.

Reply:
@oliviertassinari No worries! My question was meant to see if it's okay to move forward with the borders 😬. I mean, the no-borers route might work but I really feel like it makes for a messier and distracting UI. Especially if we add more distinctive-looking logos (Octopus and DoiT somehow match?!)

Reply:
- No objections to keep the borders.
- I think that a squared logo is not working, a rectangle one, with more space for the sponsors will likely work better.

Reply:
Any ideas on how to address the second bullet? Asking because I'm not sure we have any leverage over it. DoiT, for instance, is a logo different from Octopus and all the other ones on the Vue side nav, it's squared.

Reply:
Right, I guess we can't do much for Doit, it would only be for Octopus?